app/views/tasks/index.html.erb

```diff
@@ -152,11 +162,11 @@
         <% end %>
         <p class="fw-lighter small m-0">
           <small>
-            Posted by <%= task.user_add %> at <%= task.added.strftime('%I:%M %p %Z') %> on <%= task.added.strftime('%b %d %Y') %>
+            Posted by <%= task.user_add %> 
          </small></p>
         <p class="m-0">
           <small>
-            <%= task.description %>
+            <%= raw task.description %>
```

Comment:
be careful with `raw`, you're leaving yourself open to javascript injection and csrf attacks!  are you sure this is what you need?

Reply:
resolved!  using sanitize now and only calling it when airtable markdown processing is necessary, reducing exposure to users being able to inject malicious code